### PR TITLE
Sometimes volume alert raise with volume_name None

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bug-id: Tendrl/node-agent#828
bugzilla: 1588440

Signed-off-by: GowthamShanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

File: tendrl/integrations/gluster/alerts/__init__.py [hunk 1]
```diff
@@ -4,23 +4,28 @@
 def update_alert_count(alert):
     if alert.resource in NS.integrations.gluster.objects.VolumeAlertCounters(
             )._defs['relationship'][alert.alert_type.lower()]:
-        alert.tags['volume_id'] = find_volume_id(
+        volume = find_volume_id(
```
Comment (shtripat) [inline tendrl/integrations/gluster/alerts/__init__.py]: function can rather be called `find_volume_with_name` I feel

File: tendrl/integrations/gluster/alerts/__init__.py [hunk 2]
```diff
@@ -4,23 +4,28 @@
...
-        alert_count = int(counter_obj.alert_count)
-        if alert.severity == constants.ALERT_SEVERITY["info"]:
-            alert_count -= 1
-        elif alert.severity == constants.ALERT_SEVERITY["warning"]:
-            alert_count += 1
-        elif alert.severity == constants.ALERT_SEVERITY["critical"]:
-            alert_count += 1
-        counter_obj.alert_count = alert_count
-        counter_obj.save()
+        if volume and (
```
Comment (shtripat) [inline tendrl/integrations/gluster/alerts/__init__.py]: Can this condition be like

```
if volume and volume.vol_id not in [None, ''] and volume.name not in [None, '']
```

File: tendrl/integrations/gluster/alerts/__init__.py (hunk 1 above)
Comment (GowthamShanmugam) [reply tendrl/integrations/gluster/alerts/__init__.py]: ok, i will change this

File: tendrl/integrations/gluster/alerts/__init__.py (hunk 2 above)
Comment (GowthamShanmugam) [reply tendrl/integrations/gluster/alerts/__init__.py]: i will change this

**Comment by GowthamShanmugam** [comment]
@shtripat changes done

**Pull request merged**